docs/admin/federation/kubefed.md

```diff
@@ -0,0 +1,172 @@
+---
+assignees:
+- madhusudancs
+
+---
+Kubernetes v1.5 introduced a new command line tool called `kubefed`
+to facilitate the administration of cluster federations. `kubefed`
```

Comment:
"to facilitate the adminstration of cluster federations" -> "to help you administrate your federated clusters."